configs/milvus.yaml

```diff
@@ -528,6 +529,7 @@ common:
     threshold:
       info: 500 # minimum milliseconds for printing durations in info level
       warn: 1000 # minimum milliseconds for printing durations in warn level
+  isDisableTsMsg: false # Whether the instance disable sending ts messages
```

Comment:
comment is misleading, maybe `whether replicate ts msg`?

Reply:
This configuration means that rootcoord will not send tt msg to mq.

Reply:
the name is weird. what about `ttMsgEnabled`